common/src/main/resources/staticdata/architecture_extensions/black_planks.json

```diff
@@ -0,0 +1,13 @@
+{
+	"only_if_present": "wood_you_dye",
+	"name": "black_plank",
```

Comment:
switch back to the color names (e.g. "black") for all of these staticdata files. Leave the lang alone, it's all set.

Reply:
oh ok